### PR TITLE
doc: frequency vehicle journeys RFC

### PR DESCRIPTION
**Pull request by benoit-bst**
Add some doc about `Frequency vehicle journeys`. 
How **Kraken** works with it
[skip-ci]

### CODE REVIEW

File: documentation/rfc/frequency_vehicle_journey.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Frequency Vehicle Journeys
+
+The GTFS specification refers to [`frequencies.txt`](https://gtfs.org/reference/static#frequenciestxt)
+that represents trips wich operate on regular headways.<br/>
```
Comment (nonifier) [inline documentation/rfc/frequency_vehicle_journey.md]:
```suggestion
that represents trips which operate on regular headways.<br/>
```

File: documentation/rfc/frequency_vehicle_journey.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+This RFC describes how kraken works with it.
+
+In first, `frequencies.txt` file is readed by *fusio2ed*. For each line, a stop time list is created from
```
Comment (nonifier) [inline documentation/rfc/frequency_vehicle_journey.md]:
```suggestion
At first, `frequencies.txt` file is read by *fusio2ed*. For each line, a stop time list is created from
```

File: documentation/rfc/frequency_vehicle_journey.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+| 05:00:00     | 05:00:00       |
+| 05:10:00     | 05:10:00       |
+| 05:20:00     | 05:20:00       |
+| 05:30:00     | 05:30:00       |
+| ...          | ...            |
+| 10:00:00     | 10:00:00       |
+
+
+The generated stop times list is used to create a `Frequency Vehicle journeys`. A bool flag (`is_frequency`) is available inside
+Vehicle journeys data structure to discriminate it. When starting *Kraken*, this data is loaded inside *Raptor* cache.
```
Comment (nonifier) [inline documentation/rfc/frequency_vehicle_journey.md]:
```suggestion
Vehicle journeys data structure to discriminate it. When starting *Kraken*, this data is loaded inside the *Raptor* cache.
```

File: documentation/rfc/frequency_vehicle_journey.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+| 1       | 09:30:00     | 09:30:00       |         |               |               |             |               |                     |
+| 1       | 09:40:00     | 09:40:00       |         |               |               |             |               |                     |
+| 1       | 09:50:00     | 09:50:00       |         |               |               |             |               |                     |
+| 1       | 10:00:00     | 10:00:00       |         |               |               |             |               |                     |
+
+*Generated stop times list*
+
+| arrival_time | departure_time |
+| ------------ | -------------- |
+| 05:00:00     | 05:00:00       |
```
Comment (nonifier) [inline documentation/rfc/frequency_vehicle_journey.md]:
```suggestion
| 09:00:00     | 09:00:00       |
```
I guess this applies for all the generated stop times.

**Review by nonifier — APPROVED** [review]
very nice of you to have documented this :pray:

**Comment by kinnou02** [comment]
Except that it doesn't work like that :D 
Do you want me to take over this RFC or do you prefer to write what you understand yesterday ?

**Review by kinnou02 — DISMISSED** [review]
Except that it doesn't work like that :D
Do you want me to take over this RFC or do you prefer to write what you understood yesterday ?

File: documentation/rfc/frequency_vehicle_journey.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+
+The generated stop times list is used to create a `Frequency Vehicle journeys`. A bool flag (`is_frequency`) is available inside
+Vehicle journeys data structure to discriminate it.<br/>
+When starting *Kraken*, this data is only loaded inside the *Raptor* cache.
+This means that it is not append into *stop times* list contained inside *Vehicle Journeys*.
+
+*Frequency Vehicle journey* is not cunning with *classic stop times*.<br/>
+As a reminder, stop times are contained inside [`stop_times.txt`](https://gtfs.org/reference/static/#stop_timestxt).<br/>
+If the *trip_id* is the same, frequency parameters alter the given stop times, describ into *stop_times.txt* file.
```
Comment (AurelienLP) [inline documentation/rfc/frequency_vehicle_journey.md]:
```suggestion
If the *trip_id* is the same, frequency parameters alter the given stop times described into *stop_times.txt* file.
```

File: documentation/rfc/frequency_vehicle_journey.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+The consequence of this meld makes the list become obsolete. The values are no longer valid and become not consistent.<br/>
```
Comment (AurelienLP) [inline documentation/rfc/frequency_vehicle_journey.md]:
```suggestion
The consequence of this meld makes the list obsolete. The values are no longer valid and become not consistent.<br/>
```

**Review by AurelienLP — APPROVED** [review]

**Comment by kinnou02** [comment]
I suggested some change in a PR on this branch: https://github.com/benoit-bst/navitia/pull/6

File: documentation/rfc/frequency_vehicle_journey.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
+# Frequency Vehicle Journeys
+
+Frequencies vehicle journeys are a special case of vehicle journeys as they operate multiple time per day
+contrary to discrete vehicle journeys that operate only one time per day.
+They are used to define a template of a journey that will start it's mission at a defined interval during a
```
Comment (nonifier) [inline documentation/rfc/frequency_vehicle_journey.md]:
```suggestion
They are used to define a journey template that will start its mission at a defined interval for a
```

File: documentation/rfc/frequency_vehicle_journey.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+specified time range.
+By example a journey that serve A, B and C, with a new VJ starting from A every five minutes between 06h00 and
```
Comment (nonifier) [inline documentation/rfc/frequency_vehicle_journey.md]:
```suggestion
By example a journey that serves A, B and C, with a new VJ starting from A every five minutes between 06h00 and
```

File: documentation/rfc/frequency_vehicle_journey.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+21h00.
+
+They are quite an exception as we know very few agencies that model their networks with frequencies, but we use
+them to model some kind of on demand transport.
+
+## Input data feed
+
+The GTFS specification refers to [`frequencies.txt`](https://gtfs.org/reference/static#frequenciestxt)
+that represents trips which operate on regular headways. The NTFS is similar on that point so no distinction
```
Comment (nonifier) [inline documentation/rfc/frequency_vehicle_journey.md]:
```suggestion
that represents trips which operate on regular headways (time between trips). The NTFS is similar on that point so no distinction
```

File: documentation/rfc/frequency_vehicle_journey.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+between them will be made in this documentation.
+
+`frequencies.txt` file is read by *fusio2ed*. For each line, the corresponding vehicle journey is updated with
+*start_time*, *end_time* and *headway_secs*.
+
+## navitia model
+
+On the kraken side frequencies vehicle journeys are implemented as a sub class of `VehicleJourney` named
```
Comment (nonifier) [inline documentation/rfc/frequency_vehicle_journey.md]:
```suggestion
On the kraken side, frequencies vehicle journeys are implemented as a sub class of `VehicleJourney` named
```

File: documentation/rfc/frequency_vehicle_journey.md [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+`FrequencyVehicleJourney` that add the fields *start_time*, *end_time* and *headway_secs* to a regular vehicle
```
Comment (nonifier) [inline documentation/rfc/frequency_vehicle_journey.md]:
```suggestion
`FrequencyVehicleJourney` that adds the fields *start_time*, *end_time* and *headway_secs* to a regular vehicle
```

**Review by nonifier — APPROVED** [review]

**Pull request merged**